loginScreen/loginScreen/New Group/Controllers/LoginViewController.swift

```diff
@@ -0,0 +1,394 @@
+        textField.placeholder = "Name"
+        
+        textField.leftView = viewWithAttachedImage(withSize: sideSubviewsIndents,
+                                                   scale: 0.7,
+                                                   imageNamed: "person.png")
+        textField.rightView = viewWithAttachedImage(withSize: sideSubviewsIndents,
+                                                    scale: 0.7,
+                                                    imageNamed: "check.png")
+        textField.leftViewMode = .always
+        textField.rightViewMode = .whileEditing
+        
+        return textField
+    }()
+    
+    private lazy var passwordView: UITextField = {
+        let textField = UITextField()
+        textField.backgroundColor = .white
+        textField.textColor = .black
+        textField.layer.cornerRadius = preferredSubviewsHeight / 2
+        textField.createShadowLayer(withCornerradius: preferredSubviewsHeight / 2)
+        textField.isSecureTextEntry = true
+        textField.placeholder = "Password"
+        
+        textField.leftView = viewWithAttachedImage(withSize: sideSubviewsIndents,
+                                                   scale: 0.7,
+                                                   imageNamed: "lock.png")
+        textField.rightView = viewWithAttachedImage(withSize: sideSubviewsIndents,
+                                                    scale: 1,
+                                                    imageNamed: "unknown")
+
+        textField.leftViewMode = .always
+        textField.rightViewMode = .always
+        
+        return textField
+
+    }()
+    
+    private lazy var loginButton: UIButton = {
+        let button = UIButton()
+        button.layer.cornerRadius = preferredSubviewsHeight / 2
+        button.backgroundColor = UIColor(red: 0.44, green: 0.41, blue: 0.82, alpha: 1.00)
+        button.setTitle("Login", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.textAlignment = .center
+        button.createShadowLayer(withCornerradius: preferredSubviewsHeight / 2)
+        button.addTarget(self, action: #selector(anyAction), for: .touchUpInside)
+        return button
+    }()
+    
+    private lazy var forgotButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Forgot your password?", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.textAlignment = .center
+        button.addTarget(self, action: #selector(anyAction), for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - UI elements: bottom stacks
+    
+    private lazy var bottomStack: UIStackView = {
+        let stackView = UIStackView()
+        stackView.addSubview(signUpButton)
+        stackView.addSubview(connectWithFacebookButton)
+        stackView.addSubview(connectWithTwitterButton)
+        stackView.addSubview(separatorView)
+        return stackView
+    }()
```

Comment:
Как правило в таком замыкании мы настраиваем вью, а вот это бы я в setupHierarchy поместил
```suggestion
    private lazy var bottomStack  = UIStackView()
```